PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+
+# What you've done
+...
+
+# Why have you done it
+...
+
+# Screenshot (if visual)
+...
+
+# Any testing carried out
+...
```

Comment:
I'd suggest adding Github automatic resolver words [1] as well. Otherwise managing the issue list would quickly become a nightmare. 

[1] - https://github.com/blog/1506-closing-issues-via-pull-requests